ci_framework/roles/hive/README.md

```diff
@@ -0,0 +1,10 @@
+## Role: hive
+This role can be used to claim clusters from a clusterpool. 
+It assumes that oc_cli > 4 is installed and configured
+It assumes cifw_hive_platform variable is defined. It switches which cluster claim tasks to include based on this variavle.
+
+### Privilege escalation
+If apply, please explain the privilege escalation done in this role.
+
+### Parameters
+* `cifw_hive_platform`: Which cloud/baremetal provider to use for hive cluster claims. Defaults to openstack
```

Comment:
```
* `cifmw_use_hive`: Defaults to false.
* `cifmw_hive_platform`: (Required), the platform to install OpenShift. It
  could be a cloud provider or baremetal. Supported values are ('openstack', 
  'baremetal').
* `cifmw_hive_kubeconfig`: (Required), absolute path to the kubeconfig file to
  be used for communicating with the Hive operator.
* `cifmw_hive_namespace`: (Required), OpenShift namespace that can be used for
  creating the required resources (secrets, clusterimageset, etc.).
* `cifmw_hive_openstack_action`: The action to be performed. Support values
  are ('create', 'claim', 'unclaim' and 'delete').
* `cifmw_hive_openstack_pool_name`: The name of OCP clusterpool to be used for
  claim. Required when platform is openstack and clusterpool exists.
* `cifmw_hive_openstack_claim_name`: The name of the claim to be used for the
  claimed OCP resource in Hive.
```
Would it make sense to use the above set of arguments?

They could of type `cifmw_<role>_<plaform>_variable`